### PR TITLE
BCF 3.0: Add test case for all parts of decomposed object except one visible

### PR DESCRIPTION
**Pull request by mattikannala**
BCF 3.0: Add test case for all parts of decomposed object except one visible

### CODE REVIEW

**Review by ykulbak — APPROVED** [review]
Changed reviewed and tested on Aconex

File: Test Cases/v3.0/Visualization/All parts of decomposed object except one visible/unzipped/28942812-8783-4d92-9c54-8a8f31937975/markup.bcf [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            </File>
+        </Files>
+    </Header>
+    <Topic Guid="28942812-8783-4d92-9c54-8a8f31937975" TopicType="Error" TopicStatus="Open">
+        <ReferenceLinks/>
+        <Title>All components except one plate visible</Title>
+        <Index>1</Index>
+        <Labels/>
+        <CreationDate>2017-05-23T11:25:28+03:00</CreationDate>
+        <CreationAuthor>support@solibri.com</CreationAuthor>
```
Comment (pasi-paasiala) [inline Test Cases/v3.0/Visualization/All parts of decomposed object except one visible/unzipped/28942812-8783-4d92-9c54-8a8f31937975/markup.bcf:17]: Should we use some other email here?

File: Test Cases/v3.0/Visualization/All parts of decomposed object except one visible/unzipped/bcf.version [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Version xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" VersionId="3.0" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/buildingSMART/BCF-XML/release_3_0/Schemas/version.xsd">
+    <DetailedVersion>3.0</DetailedVersion>
```
Comment (pasi-paasiala) [inline Test Cases/v3.0/Visualization/All parts of decomposed object except one visible/unzipped/bcf.version:3]: Detailed version need to be removed.

**Review by pasi-paasiala — CHANGES_REQUESTED** [review]
-1

At least detailed version needs to go.

**Comment by leosobral** [comment]
Tested OK in Bimsync

File: Test Cases/v3.0/Visualization/All parts of decomposed object except one visible/unzipped/28942812-8783-4d92-9c54-8a8f31937975/markup.bcf (hunk 1 above)
Comment (mattikannala) [reply Test Cases/v3.0/Visualization/All parts of decomposed object except one visible/unzipped/28942812-8783-4d92-9c54-8a8f31937975/markup.bcf]: Changed to test@bcfxmltest.com

File: Test Cases/v3.0/Visualization/All parts of decomposed object except one visible/unzipped/bcf.version (hunk 2 above)
Comment (mattikannala) [reply Test Cases/v3.0/Visualization/All parts of decomposed object except one visible/unzipped/bcf.version]: Removed

File: Test Cases/v3.0/Visualization/All parts of decomposed object except one visible/unzipped/project.bcfp [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<ProjectExtension xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/buildingSMART/BCF-XML/release_3_0/Schemas/project.xsd">
+    <Project ProjectId="1e96546e-ef06-4725-81c5-8c9df7480783">
+        <Name>TestProject</Name>
+    </Project>
+    <ExtensionSchema></ExtensionSchema>
```
Comment (pasi-paasiala) [inline Test Cases/v3.0/Visualization/All parts of decomposed object except one visible/unzipped/project.bcfp:6]: This line needs to go.

**Review by pasi-paasiala — CHANGES_REQUESTED** [review]
extensions.xml needs to be added.

File: Test Cases/v3.0/Visualization/All parts of decomposed object except one visible/unzipped/project.bcfp (hunk 3 above)
Comment (mattikannala) [reply Test Cases/v3.0/Visualization/All parts of decomposed object except one visible/unzipped/project.bcfp]: The file is not more

**Comment by pasi-paasiala** [comment]
> extensions.xml needs to be added.

This is now fixed

**Review by pasi-paasiala — APPROVED** [review]
+2

**Pull request merged**